NAMESPACE

```diff
@@ -105,6 +106,7 @@ S3method(makeContent,labelgrob)
 S3method(makeContext,dotstackGrob)
 S3method(merge_element,default)
 S3method(merge_element,element)
+S3method(merge_element,element_blank)
```

Comment:
These diff seems to come from #3570, but I'm OK whether or not they are included in this PR.

Reply:
Good catch. I opened a separate PR to disentangle these, see #3595.